internal/helm/flags/flag.go

```diff
@@ -76,14 +78,16 @@ see https://github.com/kubernetes-sigs/controller-runtime/issues/895 for more in
 	// TODO(2.0.0): remove
 	flagSet.StringVar(&f.MetricsBindAddress,
 		"metrics-addr",
-		":8080",
-		"The address the metric endpoint binds to",
+		"0",
+		"The address the metrics endpoint binds to. "+
+			"Use :8443 for HTTPS or :8080 for HTTP, or leave as 0 to disable the metrics service.",
 	)
 	_ = flagSet.MarkDeprecated("metrics-addr", "use --metrics-bind-address instead")
 	flagSet.StringVar(&f.MetricsBindAddress,
 		"metrics-bind-address",
-		":8080",
-		"The address the metric endpoint binds to",
+		"0",
+		"The address the metrics endpoint binds to. "+
+			"Use :8443 for HTTPS or :8080 for HTTP, or leave as 0 to disable the metrics service.",
```

Comment:
These are breaking changes since they change the defaults. I think it's okay to leave this set at `":8080" though, and still be able to provide scaffolding in the new version that sets them how we want?

Reply:
I wasn't sure if we also considered this a breaking change, totally fine reverting this and having go/helm be different for these params.

Reply:
reverted this changes